tests/system/providers/google/cloud/ai_platform/example_mlengine.py

```diff
@@ -0,0 +1,381 @@
+    create_version = MLEngineCreateVersionOperator(
+        task_id="create-version",
+        project_id=PROJECT_ID,
+        model_name=MODEL_NAME,
+        version={
+            "name": "v1",
+            "description": "First-version",
+            "deployment_uri": f'{JOB_DIR}/keras_export/',
+            "runtime_version": "2.1",
+            "machineType": "mls1-c1-m2",
+            "framework": "TENSORFLOW",
+            "pythonVersion": "3.7",
+        },
+    )
+    # [END howto_operator_gcp_mlengine_create_version1]
+
+    # [START howto_operator_gcp_mlengine_create_version2]
+    create_version_2 = MLEngineCreateVersionOperator(
+        task_id="create-version-2",
+        project_id=PROJECT_ID,
+        model_name=MODEL_NAME,
+        version={
+            "name": "v2",
+            "description": "Second version",
+            "deployment_uri": SAVED_MODEL_PATH,
+            "runtime_version": "2.1",
+            "machineType": "mls1-c1-m2",
+            "framework": "TENSORFLOW",
+            "pythonVersion": "3.7",
+        },
+    )
+    # [END howto_operator_gcp_mlengine_create_version2]
+
+    # [START howto_operator_gcp_mlengine_default_version]
+    set_defaults_version = MLEngineSetDefaultVersionOperator(
+        task_id="set-default-version",
+        project_id=PROJECT_ID,
+        model_name=MODEL_NAME,
+        version_name="v2",
+    )
+    # [END howto_operator_gcp_mlengine_default_version]
+
+    # [START howto_operator_gcp_mlengine_list_versions]
+    list_version = MLEngineListVersionsOperator(
+        task_id="list-version",
+        project_id=PROJECT_ID,
+        model_name=MODEL_NAME,
+    )
+    # [END howto_operator_gcp_mlengine_list_versions]
+
+    # [START howto_operator_gcp_mlengine_print_versions]
+    list_version_result = BashOperator(
+        bash_command=f"echo {list_version.output}",
+        task_id="list-version-result",
+    )
+    # [END howto_operator_gcp_mlengine_print_versions]
+
+    # [START howto_operator_gcp_mlengine_get_prediction]
+    prediction = MLEngineStartBatchPredictionJobOperator(
+        task_id="prediction",
+        project_id=PROJECT_ID,
+        job_id="prediction-{{ ts_nodash }}-{{ params.model_name }}",
+        region=REGION,
+        model_name=MODEL_NAME,
+        data_format="TEXT",
+        input_paths=[PREDICTION_INPUT],
+        output_path=PREDICTION_OUTPUT,
+        labels={"job_type": "prediction"},
+    )
+    # [END howto_operator_gcp_mlengine_get_prediction]
+
+    # [START howto_operator_gcp_mlengine_delete_version]
+    delete_version = MLEngineDeleteVersionOperator(
+        task_id="delete-version", project_id=PROJECT_ID, model_name=MODEL_NAME, version_name="v1"
+    )
+    # [END howto_operator_gcp_mlengine_delete_version]
+
+    # [START howto_operator_gcp_mlengine_delete_model]
+    delete_model = MLEngineDeleteModelOperator(
+        task_id="delete-model", project_id=PROJECT_ID, model_name=MODEL_NAME, delete_contents=True
+    )
```

Comment:
If this model is stored in the service we should clean it up after the test with trigger_rule.ALL_DONE. Since it's inside the documentation markers (..# [START) we can do it outside the markers:

```
    # [START howto_operator_gcp_mlengine_delete_model]
    delete_model = MLEngineDeleteModelOperator(
        task_id="delete-model", project_id=PROJECT_ID, model_name=MODEL_NAME, delete_contents=True
    )
   # [END howto_operator_gcp_mlengine_delete_model]
   delete_model.trigger_rule = TriggerRule.ALL_DONE
```